apisix/plugins/prometheus/exporter.lua

```diff
@@ -351,6 +356,42 @@ local function etcd_modify_index()
 
 end
 
+local key = {}
+local combine_param = {}
```

Comment:
can we init `key` and `combine_param` as table if `header_of_shared_dict` is true?

Reply:
> can we init `key` and `combine_param` as table if `header_of_shared_dict` is true?

I consider the problem of repeated malloc for table `key` and `combine_param`.

Reply:
Do you mean that you avoid creating tables every time? Sounds reasonable.